### PR TITLE
I didn't push after changing the module name so here goes

### PR DESCRIPTION
**Pull request by eahefnawy**

### CODE REVIEW

File: yellow/exceptions.py [hunk 1]
```diff
@@ -1,2 +0,0 @@
-class YellowApiError(Exception): pass
-class YellowRequestError(Exception): pass
```
Comment (JamesPiechota) [inline yellow/exceptions.py:2]: Ha. Good call.

**Comment by JamesPiechota** [comment]
LGTM

**Comment by eahefnawy** [comment]
I fixed the verify function to be request agnostic, and also, the create and query functions now return dict instead of response object.

**Comment by JamesPiechota** [comment]
LGTM. These changes have already been uploaded to pypi, right?

**Comment by eahefnawy** [comment]
yep uploaded and signed ;)

**Pull request merged**